resources.whatwg.org/build/review.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-set -o errexit
```

Comment:
Why remove this? If it's for diff, then I think a better solution would be to add `|| 0` to the end of that line

Reply:
When I try that on whatwg/html's equivalent I get `./review-draft.sh: line 26: 0: command not found`.

Reply:
I reckon Domenic meant `|| exit 0`

Reply:
It looks like `true` is what I meant. I will add suggested changes here.